lerna.json

```diff
@@ -0,0 +1,19 @@
+{
+  "packages": [
+    "sandpack-client",
+    "sandpack-react"
+  ],
+  "npmClient": "yarn",
+  "version": "independent",
```

Comment:
Not sure if this is an intentional change, but I second the decision to not have the same version for the react and the client libraries